README.md

```diff
@@ -60,6 +60,40 @@ through [the behave
 tutorial](https://behave.readthedocs.io/en/latest/tutorial.html) to get
 an idea of how it works, and how tests are written.)
 
+#### Iterating Locally
+
+To make running the tests repeatedly less time-intensive, our behave
+testing setup has support for reusing images between runs via two
+configuration options (provided in environment variables),
+`UACLIENT_BEHAVE_IMAGE_CLEAN` and `UACLIENT_BEHAVE_REUSE_IMAGE`.
+
+To avoid the test framework cleaning up the image it creates, you can
+run it like this:
+
+```sh
+UACLIENT_BEHAVE_IMAGE_CLEAN=0 behave
+```
+
+which will emit a line like this above the test summary:
+
+```
+Image cleanup disabled, not deleting: behave-image-1572443113978755
+```
+
+You can then reuse that image by plugging its name into your next test
+run, like so:
+
+```sh
+UACLIENT_BEHAVE_REUSE_IMAGE=behave-image-1572443113978755 behave
+```
+
+If you've done this correctly, you should see test execution start
```

Comment:
Could note you will see the config option reporting " reuse_image = <YOUR_IMAGE_NAME>"